### PR TITLE
Bound more ClientHandle functions to lua

### PR DESCRIPTION
**Pull request by AirOne01**
A demo plugin to test these is available [here](https://gist.github.com/AirOne01/3334c2d358f7cb0f460bcdbb1d7cd405)
* Bound the following functions to lua
  + SendBlockBreakAnim
  + SendCollectEntity
  + SendDestroyEntity
  + SendDetachEntity
  + SendEditSign
  + SendExperienceOrb
  + SendLeashEntity
  + SendThunderbolt
  + SendUnleashEntity

### CODE REVIEW

**Comment by AirOne01** [comment]
Refer to #5263

**Comment by AirOne01** [comment]
I forgot to update the doc. I will do that later after I catch some sleep.

File: Server/Plugins/APIDump/APIDesc.lua [hunk 1]
```diff
@@ -1542,6 +1542,33 @@ end
 					},
 					Notes = "Kicks the user with the specified reason",
 				},
+        SendBlockBreakAnim =
```
Comment (bearbin) [inline Server/Plugins/APIDump/APIDesc.lua:1545]: Your indentation here is wrong, you need to be using tabs rather than spaces.

**Review by bearbin — COMMENTED** [review]
Looks good, thanks for the work! You need to fix the indentation, and then add your name to the CONTRIBUTORS file so your contribution is recognised and you show you accept the license; then this should be good to merge in.

**Comment by AirOne01** [comment]
> Looks good, thanks for the work! You need to fix the indentation, and then add your name to the CONTRIBUTORS file so your contribution is recognised and you show you accept the license; then this should be good to merge in.

Done !

File: CONTRIBUTORS [hunk 2]
```diff
@@ -8,6 +8,7 @@ as provided in the LICENSE file.
 12xx12
 9caihezi
 Altenius
+AirOne01
```
Comment (bearbin) [inline CONTRIBUTORS:11]: i comes before L in the alphabet :)

Comment (AirOne01) [reply CONTRIBUTORS:11]: this is a bruh moment, thanks 0_0

**Review by bearbin — APPROVED** [review]

**Comment by tigerw** [comment]
Nice collab
`Co-authored-by: BuildTools <unconfigured@null.spigotmc.org>`

**Comment by AirOne01** [comment]
> Nice collab `Co-authored-by: BuildTools <unconfigured@null.spigotmc.org>`

No idea how this happened, as I literraly configured git globally and it worked for other projects.

**Pull request merged**